kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_componentwise.h

```diff
@@ -370,9 +370,7 @@ class ResidualBasedEliminationBuilderAndSolverComponentwise
         TSystemMatrixPointerType& pA,
         TSystemVectorPointerType& pDx,
         TSystemVectorPointerType& pb,
-        ElementsArrayType& rElements,
-        ConditionsArrayType& rConditions,
-        ProcessInfo& CurrentProcessInfo
+        ModelPart& rModelPart
     ) override
     {
 #ifndef __SUNPRO_CC
```

Comment:
this ifndef is legacy, we could take it out.

Reply:
I took it away.